sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/cli/CommandLineParser.groovy

```diff
@@ -0,0 +1,39 @@
+package life.qbic.samplenotificator.cli
+
+import groovy.util.logging.Log4j2
+import picocli.CommandLine
+
+/**
+ * <h1>Parses the command line</h1>
+ *
+ * <p>A parser for processing the commandline input.</p>
+ *
+ * @since 1.0.0
+ *
+*/
+@Log4j2
+class CommandLineParser {
```

Comment:
I am not sure if it is an idea to replace this with https://github.com/qbicsoftware/core-utils-lib/blob/master/src/main/java/life/qbic/cli/ToolExecutor.java